tests/poll.rs

```diff
@@ -201,10 +208,112 @@ fn registry_behind_arc() {
     handle2.join().unwrap();
 }
 
+// Testing on Windows only where this must fail by design, on other platforms
```

Comment:
Nit: Comment style is pretty different than the rest of Mio. I'd prefer something like:

```
// Testing on Windows ..
//
// This test checks the following register/deregister ..
```

And the remove the `/* .. */` comment block inside.

Reply:
ok, will redo the comments style